dora/stress/shell/src/main/java/alluxio/stress/cli/worker/StressWorkerBench.java

```diff
@@ -465,65 +464,81 @@ private void runInternal() throws Exception {
       WorkerBenchCoarseDataPoint dp = new WorkerBenchCoarseDataPoint(
           Long.parseLong(workerID),
           Thread.currentThread().getId());
-      int sliceCount = 0;
-      int sliceIoBytes = 0;
+      WorkerBenchDataPoint slice = new WorkerBenchDataPoint();
       List<Long> throughputList = new ArrayList<>();
-      int lastSlice = 0;
+      long lastSlice = 0;
 
       while (!Thread.currentThread().isInterrupted()
           && CommonUtils.getCurrentMs() < mContext.getEndMs()) {
         // Keep reading the same file
         long startMs = CommonUtils.getCurrentMs() - recordMs;
-        long bytesRead = applyOperation();
-        long duration = CommonUtils.getCurrentMs() - recordMs - startMs;
+        ApplyOperationOutput fileReadOutput = applyOperation();
         if (startMs > 0) {
-          if (bytesRead > 0) {
-            mResult.setIOBytes(mResult.getIOBytes() + bytesRead);
-            sliceCount += 1;
-            sliceIoBytes += bytesRead;
-            // if duration is 0ms, treat is as 1ms for now
-            if (duration == 0) {
-              throughputList.add(bytesRead);
+          if (fileReadOutput.mBytesRead > 0) {
+            mResult.setIOBytes(mResult.getIOBytes() + fileReadOutput.mBytesRead);
+            slice.mCount += 1;
+            slice.mIOBytes += fileReadOutput.mBytesRead;
+            if (fileReadOutput.mDuration > 0) {
+              // throughput unit: MB/s
+              // max file size allowed: 9223372036B (8.5GB)
+              throughputList.add(fileReadOutput.mBytesRead * 1000000000
```

Comment:
In java a syntactic sugar is you can use underscore in a number to improve readability:
`1000000000` -> `1_000_000_000`

This kind of constants are defined in `alluxio.Constants`. You can actually find your `SECOND_NANO` and `MB` there.